src/otx/algo/action_detection/x3d_fastrcnn.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""X3DFastRCNN model implementation."""
+from __future__ import annotations
+
+from otx.algo.utils.mmconfig import read_mmconfig
+from otx.core.model.entity.action_detection import MMActionCompatibleModel
+
+
+class X3DFastRCNN(MMActionCompatibleModel):
+    """X3D Model."""
+
+    def __init__(self, num_classes: int, topk: int | tuple[int]):
```

Comment:
here `topk` is defined as an argument, which is different than the previous X3D signature. does the difference stem from the task type, or something else?

Reply:
This PR aims to introduce a dedicated Python class for each OTX model. Therefore, the efforts have been made to apply minimal changes whenever possible. `topk` is a configurable parameter in the integration tests, so I added it here to ensure compatibility with the integration test. @jaegukhyun, please see this and clean up the input arguments of this model if possible

Reply:
In this structure, topk can be configured by num_classes, I'll update it after merging this PR.